src/tags/object/RichText/domManager.ts

```diff
@@ -1,34 +1,51 @@
 import { flatten, isDefined } from '../../../utils/utilities';
 
+// line feed
+const LF = '\n';
```

Comment:
```suggestion
const LINE_FEED = '\n';
```
It would be better to qualify the constant instead of adding a comment above the abbreviated as this is then clear at point of usage.